### ISSUE TITLE
import to integer field seems to result in incorrect values

### ISSUE THREAD

**Issue by travisturner** [pilosa/pilosa#2083]
### What's going wrong?
Integer values imported with `pilosa import` seem to be getting set incorrectly when they come after a `Row()` query. I haven't dug into this, but I wanted to report it as it seems to be a problem. See the example below for what's happening.

### What was expected?
Separate imports result in the same values as a single import.

### Steps to reproduce the behavior

This works as expected:
```
curl localhost:10101/index/t -d '{"options": {"keys": false, "trackExistence": false}}'
curl localhost:10101/index/t/field/f -d '{"options": {"type": "int", "keys": false}}'
echo "2,1" > tmp.csv; pilosa import -i t -f f tmp.csv
echo "1000,1" > tmp.csv; pilosa import -i t -f f tmp.csv
curl localhost:10101/index/t/query -d 'Row(f>0)'
```
The result is:
```
{"results":[{"attrs":{},"columns":[2,1000]}]}
```

However, when a `Row` query is run between the two imports, the second import seems to get applied incorrectly.
```
curl localhost:10101/index/t2 -d '{"options": {"keys": false, "trackExistence": false}}'
curl localhost:10101/index/t2/field/f2 -d '{"options": {"type": "int", "keys": false}}'
echo "2,1" > tmp.csv; pilosa import -i t2 -f f2 tmp.csv
curl localhost:10101/index/t2/query -d 'Row(f2>0)'
echo "1000,1" > tmp.csv; pilosa import -i t2 -f f2 tmp.csv
curl localhost:10101/index/t2/query -d 'Row(f2>0)'
curl localhost:10101/index/t2/query -d 'Row(f2<0)'
```
The result here is:
```
{"results":[{"attrs":{},"columns":[2]}]}
{"results":[{"attrs":{},"columns":[1000]}]}
```

### Information about your environment (OS/architecture, CPU, RAM, cluster/solo, configuration, etc.)

For feature requests, please provide the following:

**Comment by david-cromberge** [comment]
Hi Pilosa team, thanks for open sourcing Pilosa, it is a great product.

To add to the above, I'm also experiencing similar behaviour on 1.4.0.

I have an integer field that contains timestamps, that I am constantly importing new values into, and the behaviour is the same - i.e. once the first query succeeds, subsequent queries do not reflect the latest values.  **When Pilosa is shut down and restarted, the new fields are now visible in queries.**

Is the scenario above supported? Is the index meant to remain static after initial importing?

As a workaround, it is possible for me to index dates as strings but this is very cumbersome.

**Comment by travisturner** [comment]
@david-cromberge I just pushed some related fixes to master via PR #2084. If you're able to test with that, let me know if you're still seeing problems on integer fields. Thanks!

**Comment by david-cromberge** [comment]
@travisturner thank you for fixing this bug so quickly, apologies for the late follow up.  I have downloaded and built the latest version, and confirms that the bug is now fixed.
